src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -285,29 +285,29 @@ The following are an example `config.toml` files:
    type="MultisigSigner"
 
    [[handlers]]
-   chain_name="avalanche"
-   chain_rpc_url="https://avalanche-fuji-c-chain-rpc.publicnode.com"
-   cosmwasm_contract="axelar1ty7mx0cllgz8xuvhn2j7e3qy8999ssgmjtktyqetl335em0g88lq6rjhl2"
+   chain_name="ethereum-sepolia"
+   chain_rpc_url="https://rpc.ankr.com/eth_sepolia"
```

Comment:
i've had some not so great experiences with ankr maybe suggest this endpoint?
https://ethereum-sepolia-rpc.publicnode.com

Reply:
This is coming from our canonical data set: https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-verifiers.json#L188